### ISSUE TITLE
Add tutorials for standard pycytominer workflows

### ISSUE THREAD

**Issue by gwaybio** [cytomining/pycytominer#223]
This was originally proposed in https://github.com/cytomining/pycytominer/issues/195#issuecomment-1217265110, but we've been thinking about adding this for quite some time.

Traditionally, we point prospective users to existing, publicly-available examples of pycytominer workflows:

- LINCS Cell Painting https://github.com/broadinstitute/lincs-cell-painting/blob/master/profiles/profile_cells.py
- Cell Health https://github.com/broadinstitute/cell-health/blob/master/1.generate-profiles/generate_profiles.py
- Profiling Resistance Mechanisms https://github.com/broadinstitute/profiling-resistance-mechanisms/blob/master/0.generate-profiles/generate-profiles.py
- And, most often, the profiling handbook https://github.com/cytomining/profiling-handbook

(and there are likely others)

We are also actively working on two workflow systems:

- Profiling recipe: https://github.com/cytomining/profiling-recipe
- Cytosnake: https://github.com/WayScience/CytoPipe

In other words, the most common pycytominer usage will be in a workflow language rather than direct calls to the tool.

Nevertheless, we should work towards making this more obvious to potential users.

**Comment by gwaybio** [comment]
Closed by #676 #681 #684
